tutorials/units-and-integration/units-and-integration.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using `scipy.integrate`\n",
+    "\n",
+    "This module will teach you how to integrate in the `astropy` and scientific python environment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Planck function\n",
+    "\n",
+    "The Planck function describes how a black-body radiates energy. We will explore how to find bolometric luminosity using the Planck function in both frequency and wavelength space."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from scipy import integrate\n",
+    "from astropy.modeling.blackbody import blackbody_lambda, blackbody_nu, BlackBody1D\n",
+    "from astropy import units as u, constants as c\n",
```

Comment:
please set the matplotlib display to inline before importing matplotlib:

```
%matplotlib inline
```

without this, the plots are displayed in a separate window (that is not in the browser).